<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>aboutDialog</class>
 <widget class="QDialog" name="aboutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>357</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>357</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>357</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>About Skill Check</string>
  </property>
  <property name="windowIcon">
   <iconset resource="skillcheck.qrc">
    <normaloff>:/images/sc_icon_16x16.png</normaloff>:/images/sc_icon_16x16.png</iconset>
  </property>
  <widget class="QPushButton" name="aboutOKButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="bgArtLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="skillcheck.qrc">:/images/sc_dialog.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>321</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Skill Check for Windows 32-bit&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Version: Mongoose Traveller Edition&lt;/p&gt;&lt;p&gt;Build: 0.7.6 (Beta)&lt;/p&gt;&lt;p&gt;Produced by Shonner Corp. Copyright (C) 2014. Visit us on the web at &lt;a href=&quot;http://www.shonner.com&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;www.shonner.com&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p&gt;Software design by Shawn Driscoll. Contact him at &lt;a href=&quot;mailto:shawndriscoll@hotmail.com?subject=Skill Check for Mongoose Traveller&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;shawndriscoll@hotmail.com&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p&gt;Qt GUI Toolkit is copyright (C) 2009 Nokia Corporation&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;The Traveller game in all forms is owned by Far Future Enterprises. Copyright 1977 - 2014 Far Future Enterprises. Traveller is a registered trademark of Far Future Enterprises.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>210</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="skillcheck.qrc"/>
 </resources>
 <connections/>
</ui>
